pages/contribute.js

```diff
@@ -273,10 +273,15 @@ class TiersPage extends React.Component {
                       <H2 fontWeight="normal" mb={2}>
                         {title}
                       </H2>
-                      {LoggedInUser?.canEditCollective(collective) && verb === 'events' && (
-                        <Link href={`/${collective.slug}/events/new`}>
+                      {LoggedInUser?.canEditCollective(collective) && (verb === 'events' || verb === 'projects') && (
+                        <Link href={`/${collective.slug}/${verb}/new`}>
                           <StyledButton buttonStyle="primary">
-                            <FormattedMessage id="event.create.btn" defaultMessage="Create Event" />
+                            {verb === 'events' && (
+                              <FormattedMessage id="event.create.btn" defaultMessage="Create Event" />
+                            )}
+                            {verb === 'projects' && (
+                              <FormattedMessage id="project.create.btn" defaultMessage="Create Project" />
```

Comment:
We already have a `SectionProjects.CreateProject` string? Could you check for duplicates when introducing new strings?

Reply:
Ops, didn't thought about that. Will do so in the future. I've corrected this. 👍🏽